apps/common/controllers/add-another-address-loop.js

```
@@ -0,0 +1,51 @@
+'use strict';
+
+const _ = require('lodash');
+const BaseController = require('./base');
+
+module.exports = class AddAnotherAddressController extends BaseController {
+  constructor(options) {
+    super(options);
+    if (options.locals.field) {
+      this.field = options.locals.field;
+    } else {
+      // eslint-disable-next-line no-console
+      console.error('No field set in locals');
```

what happens if options.locals.field isn't set? Can it be omitted? if not then this should actually throw an error